bindings/pyroot/ROOT.py

```diff
@@ -427,6 +427,8 @@ def __getattr1( self, name ):             # "start-up" getattr
    def __getattr2( self, name ):             # "running" getattr
     # handle "from ROOT import *" ... can be called multiple times
       if name == '__all__':
+         if sys.hexversion >= 0x3060000:
+            raise ImportError('"from ROOT import *" is not supported in Python 3')
```

Comment:
Hi Martin,
We should perhaps write "not supported in Python>=3.6" in the error message.

Reply:
As the statement in the JIRA issue was that it "does not work [reliably] in python 3" I chose to be more general in the error message. Apart from us I don't think there are many people using ROOT and python3 yet. But I have no problem to change this to the more specific message you suggest.

Reply:
We can disable it for all Python3, but then the condition check needs to be modified accordingly. Thanks Martin!